dspace-api/src/main/java/org/dspace/statistics/SolrLoggerServiceImpl.java

```diff
@@ -772,83 +768,40 @@ public void process(SolrInputDocument doc) throws IOException, SolrServerExcepti
         }
     }
 
-
     @Override
-    public void markRobotsByIP() {
-        for (String ip : SpiderDetector.getSpiderIpAddresses()) {
-
-            try {
-
-                /* Result Process to alter record to be identified as a bot */
-                ResultProcessor processor = new ResultProcessor() {
-                    @Override
-                    public void process(SolrInputDocument doc) throws IOException, SolrServerException {
-                        doc.removeField("isBot");
-                        doc.addField("isBot", true);
-                        solr.add(doc);
-                        log.info("Marked " + doc.getFieldValue("ip") + " as bot");
-                    }
-                };
-
-                /* query for ip, exclude results previously set as bots. */
-                processor.execute("ip:" + ip + "* AND -isBot:true");
-
-                solr.commit();
-
-            } catch (Exception e) {
-                log.error(e.getMessage(), e);
-            }
-
-
-        }
-
-    }
-
-    @Override
-    public void markRobotByUserAgent(String agent) {
-        try {
-
-            /* Result Process to alter record to be identified as a bot */
-            ResultProcessor processor = new ResultProcessor() {
-                @Override
-                public void process(SolrInputDocument doc) throws IOException, SolrServerException {
+    public void markRobots() {
+        ResultProcessor processor = new ResultProcessor() {
```

Comment:
In reviewing this PR, I'm wondering why the entire `SpiderDetector.getSpiderIpAddresses()` method (replaced on this line) isn't removed? That method seems to no longer be called except by Tests.

Based on the purpose of this PR, it seems like the PR should **also** be deleting all the code related the (outdated) `iplists.com-*.txt` files in this folder https://github.com/DSpace/DSpace/tree/main/dspace/config/spiders   But, maybe I'm misunderstanding the scope of this PR?

Is there are reason to keep around the IP address code when you are replacing its usages?

Reply:
You're right.  Removed.